api/proto/config/devcommon.proto

```diff
@@ -49,4 +49,14 @@ message ConfigItem {
 message Adapter {
   org.lfedge.eve.common.PhyIoType type = 1;
   string name = 2;	// Short hand name such as "com" from bundle
+  EthVF eth_vf = 3; // Only relevant for PHY_IO_TYPE_PHY_IO_NET_ETH_VF
 }
+
+// Information regarding Virtual Function (VF) customisation
+message EthVF {
+  uint32 index = 1;
+  string pci = 2;
+  string mac = 3;
+  uint32 vlanId = 4;
```

Comment:
if you change this to vlan_id then yetus will be quiet and the generate .go files will not change (the protoc will make the golang struct have vlanId).